io/io.h

```diff
@@ -21,6 +21,10 @@ inline Bytes Buffer(std::string_view str) {
   return Bytes{reinterpret_cast<const uint8_t*>(str.data()), str.size()};
 }
 
+template<size_t N> inline MutableBytes MutableBuffer(char (&buf)[N]) {
+  return MutableBytes{reinterpret_cast<uint8_t*>(buf), N};
+}
```

Comment:
accepting absl span was also an option, but I see this code is used only for tests